server/plugin.go

```diff
@@ -48,19 +61,18 @@ func (p *Plugin) OnActivate() error {
 		return err
 	}
 
-	split := strings.Split(p.configuration.TeamChannel, ",")
-	if len(split) != 2 {
+	teamChannels, err := parseTeamChannelsNames(p.configuration.TeamChannel)
```

Comment:
Unrelated to the changes in this PR:

It's concerning that this is all happening in `OnActivate`, and not also triggered by `OnConfigurationChange`, since I believe this means that you have to restart the plugin for any config changes to take effect.

Reply:
Created a ticket for this https://github.com/mattermost/mattermost-plugin-aws-SNS/issues/77